src/generate-events.js

```diff
@@ -0,0 +1,25 @@
+import { readFileSync } from "fs";
+import got from "got";
+import { parse } from "node-html-parser";
+import { resolve } from "path";
+
+import { paths } from "./utils.js";
+
+const MHNNewsUrl = "https://monsterhunternow.com/news";
+
+const getHTMLFixture = () => {
+  const data = readFileSync(resolve(paths.fixtures, "news-index.html"), {
+    encoding: "utf8",
+    flag: "r",
+  });
+  return data;
+};
```

Comment:
Temporary fixture to test against instead of constantly pulling the source contents.